jobs/scrape_archives.py

```diff
@@ -43,12 +45,17 @@ def archive_url(product, version=None, platform=None, locale=None):
     return url
 
 
+@backoff.on_exception(backoff.expo,
+                      (aiohttp.ClientError,
+                       asyncio.TimeoutError,
+                       concurrent.futures.CancelledError),
+                      max_tries=8)
```

Comment:
you're right we could get rid of the backoff for kinto operations since we already have retry in client